File: mybin/se/sysconfig/formatter/java/examples/catchstmt.java
class Statement {
   private boolean safe_foo(float dv)throws UncaughtException {
      try {
         foo(dv);
         return true;
      }
      catch (IOException|FileNodeFoundException e) {
         return false;
      }
      try (SomeHandle h = acquireThingy(dv)) {
         doSomething(h)
      }
   }
}

File: mybin/se/sysconfig/formatter/cs/examples/catchstmt.cs
class Statement {
   private boolean safe_foo(float dv) {
      try {
         foo(dv);
         return true;
      }
      catch (System.DivideByZeroException& e) {
         return false;
      }
      finally { 
         log_something();
      }
   }
}
